images/docker-compose.yml

```diff
@@ -0,0 +1,128 @@
+version: "3"
```

Comment:
Why do we need to copy blockchain-ops to here? is it for tests? can't we clone it in test from there?

Reply:
Blockchain ops include other things that are not needed, such as laboratory and archivist.
Also, the docker-compose files in blockchain-ops are not version-locked, so any changes there will also affect the tests here.

Also, I can't respond to the first comment you left about the changelog so ill do it here:

12. Decimal precision would be important when choosing an amount to send, however, both the original kin-core and py-stellar-base use floats for the amounts that you send.
I made sure to check to raise an exception for an amount that is too precise, instead of rounding it.

The only place Decimals where used in the balance of an account, this would provide no gain, and only cause annoying stuff like

```python
balance = sdk.get_kin_balance()
balance +-*/ 2 >> raise errors since you cant operate with Decimal and Float/Int
```

doing this would provide no gain:
```python
from decimal import Decimal
balance = sdk.get_kin_balance()
balance -+*/ Decimal(2)


20. The sdk is meant for the devs who will not use costume assets, and removing this ability allows me to simplify in some places, like not asking asset codes, or not specifying issuers in tx data

Reply:
well cloning for tests additional files is not a real concern, but the version-locked it is, does it mean that even when checkout a specific version it can be changed over time? (in android I did it like this)

As for Float, great if you did the homework then all good